app/main/posts/views/post-view-map.controller.js

```diff
@@ -0,0 +1,191 @@
+module.exports = ['PostEndpoint', 'Maps', '_', 'PostFilters', 'Leaflet', '$q', '$rootScope', '$compile', '$location', '$timeout', '$scope',
```

Comment:
I think this is accidentally reverting changes from the main branch?

Reply:
yep, true